app/views/candidate_interface/application_choices/review.html.erb

```diff
@@ -20,31 +20,20 @@
   </div>
 </div>
 
-<%= form_with model: @application_form, url: candidate_interface_course_choices_complete_path do |f| %>
-  <%= f.govuk_error_summary %>
-
-  <%= render(CandidateInterface::CourseChoicesReviewComponent.new(application_form: @application_form)) %>
-
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <% if @application_choices.present? %>
-        <% if @application_choices.count >= 1 %>
-          <%= f.govuk_check_boxes_fieldset :course_choices_completed, legend: nil do %>
-            <%= f.hidden_field :course_choices_completed, value: false %>
-            <%= f.govuk_check_box(
-              :course_choices_completed,
-              true,
-              multiple: false,
-              label: {
-                text: t('application_form.courses.complete.completed_checkbox'),
-              },
-              link_errors: true,
-            ) %>
-          <% end %>
-
-          <%= f.govuk_submit t('continue') %>
-        <% end %>
-      </div>
-    </div>
-  <% end %>
-<% end %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% if @application_choices.present? &&  @application_choices.count >= 1 %>
+      <%= render(CandidateInterface::CompleteSectionComponent.new(
+        section_complete_form: @section_complete_form,
+        path: candidate_interface_course_choices_complete_path,
+        request_method: :patch,
+        summary_component: CandidateInterface::CourseChoicesReviewComponent.new(application_form: @application_form),
+        complete_hint_text: if @application_choices.count == 1
+                              t('application_form.courses.complete.hint_text_two_courses')
+                            elsif @application_choices.count == 2
+                              t('application_form.courses.complete.hint_text_one_course')
+                            end,
+      )) %>
```

Comment:
This is the only case where the hint text depends on some sort of condition. I don't think this logic should sit inside the component as it is very specific, however it doesn't feel right leaving it here either. Would it be best placed in a controller instance var or helper?

Reply:
@TobyRet I think you might be able to tidy this up by using the [pluralization feature](https://guides.rubyonrails.org/i18n.html#pluralization) from Rails’s I18n gem?

Something like:

```ruby
<%= t('application_form.courses.complete.hint_text', count: @application_choices.count)
```

And then in the locale file:

```yml
hint_text:
  one: You can add one more course
  other: You can add %{count} more courses
```

Reply:
Ooooh, 👌

Reply:
Nice 👍